### ISSUE TITLE
Add title and caption

### ISSUE THREAD

**Issue by annProg** [derhuerst/svg-radar-chart#13]
It will be perfect if this project can support the title and the legend , as in the picture below

![wc9b1](https://user-images.githubusercontent.com/4145852/36072747-f7f49f3e-0f5f-11e8-9e04-0113186a2c55.jpg)

**Comment by derhuerst** [comment]
I agree that the legend would be nice to have. Keep in mind that this is not the easiest task with SVG, because it has AFAIK no automatic line wrapping, and necause the legend might easily extend beyond the area of the image.
